### ISSUE TITLE
Add viewer for Bing Streetside and Mapilary

### ISSUE THREAD

**Issue by upintheairsheep** [bryceco/GoMap#697]
Hello, can you add the ability to view street level imagery for adding things like businesses that aren’t there and adding minor features like fire hydrants and whatnot? Bing’s Ariel imagery licence also extends to streetside and Mapilary is all public imagery.

**Comment by bryceco** [comment]
Looking for more feedback on this. My first impulse is to close this as out-of-scope for a mobile editor, because you should be on location. But I accept (and appreciate!) that some people use it for armchair mapping so I'd like to hear opinions on how broadly this feature would be used.

**Comment by upintheairsheep** [comment]
> Looking for more feedback on this. My first impulse is to close this as out-of-scope for a mobile editor, because you should be on location. But I accept (and appreciate!) that some people use it for armchair mapping so I'd like to hear opinions on how broadly this feature would be used.

As an example, it could be used to properly identify unclear features, and Mapilliary provides AI object recognition to assist in mapping items

**Comment by dieterdreist** [comment]
sent from a phone

> On 1 Jul 2024, at 15:45, upintheairsheep ***@***.***> wrote:
> 
> Looking for more feedback on this. My first impulse is to close this as out-of-scope for a mobile editor, because you should be on location.


I agree this should be out of scope

**Comment by tordans** [comment]
Adding Mapillary to GoMap would be a major project with many edge cases that need to be solved. And I don't see the benefit of looking at Mapillary lines and pictures. I agree that the focus should be mobile mapping.

However, there are cases that I find relevant for precisely this reason of optimizing for mobile mapping…

* Taking pictures of object that I see next to me; MapComplete is planning an integration with Panoramax to allow geo located images. (Right now it uses some other server which we should not use IMO.) Once that feature is ready, it would be great to be able to add images to Panoramax and link them to the given objects in GoMap
* Validating detected features from Mapillary. The detections are often not ideal and need to be verified. This is a lot easier when on the ground. This is part of the reason why I am exciting for the GeoJSON support, because it allows to hand craft file that can be verified on the ground.

Both topics are for other issues, but I thought I paint a bigger picture here :). HTH

**Comment by 1ec5** [comment]
Does Go Map!! have many iPad users? Those are sometimes used while sitting in an armchair.
